app/templates/views/get-started.html

```diff
@@ -0,0 +1,80 @@
+{% extends "content_template.html" %}
+{% from "components/table.html" import mapping_table, row, text_field, edit_field, field with context %}
+{% from "components/sub-navigation.html" import sub_navigation %}
+
+{% block per_page_title %}
+  Get started
+{% endblock %}
+
+{% block content_column_content %}
+
+<h1 class="heading-large">Get started</h1>
+
+<ol class="list list-number">
+  <li>
+    <h2 class="heading-small">Check if GOV.UK Notify is right for you</h2>
+    <p>Read about our <a href="{{ url_for('main.features') }}">features</a>, <a href="{{ url_for('.pricing') }}">pricing</a> and <a href="{{ url_for('main.roadmap') }}">roadmap</a>.</p>
+    <details>
+      <summary>Organisations that can use Notify</summary>
+      <div id="eligible-organisations">
+        <p>Notify is available to:</p>
+        <ul class="list list-bullet">
+          <li>central government departments</li>
+          <li>local authorities</li>
+          <li>state-funded schools</li>
+          <li>housing associations</li>
+          <li>the NHS</li>
+          <li>companies owned by local or central government that deliver services on their behalf</li>
+        </ul>
+        <p>Notify is not currently available to charities.</p>
+      </div>
+    </details>
+  </li>
+
+  <li>
+    <h2 class="heading-small">Create an account</h2>
+    {% if not current_user.is_authenticated %}
+      <p><a href="{{ url_for('.register') }}">Create an account</a> for free and add your first Notify service. When you add a new service, it will start in <a href="{{ url_for('main.trial_mode_new') }}">trial mode</a>.</p>
+    {% else %}
+      <p>Create an account for free and add your first Notify service. When you add a new service, it will start in <a href="{{ url_for('main.trial_mode_new') }}">trial mode</a>.</p>
+    {% endif %}
+  </li>
+
+  <li>
+    <h2 class="heading-small">Write some messages</h2>
+    <p>Add <a href="{{ url_for('main.features', _anchor='templates') }}">message templates</a> with examples of the content you plan to send.<!-- You do not need to add letter templates if you’re planning to <a href="{{ url_for('main.features_letters', _anchor='upload-letters') }}">upload your own letters</a>.--></p>
+    <p>The GOV.UK Service Manual has advice on:</p>
+    <ul class="list list-bullet">
+      <li><a href="https://www.gov.uk/service-manual/design/sending-emails-and-text-messages#how-to-write-emails-and-text-messages">how to write emails and text messages</a></li>
+      <li><a href="https://www.gov.uk/service-manual/design/writing-effective-letters">writing effective letters</a></li>
+    </ul>
+  </li>
+
+  <li>
+    <h2 class="heading-small">Set up your service</h2>
+    {% if not current_user.is_authenticated %}
```

Comment:
If they go straight to this page after logging in they might not have a most-recently-view service, so you need to guard against that as well.
```suggestion
    {% if not current_user.is_authenticated or not current_service %}
```

Reply:
👍